src/components/views/context_menus/MessageContextMenu.js

```diff
@@ -88,6 +88,14 @@ module.exports = React.createClass({
         if (this.props.onFinished) this.props.onFinished();
     },
 
+    onForwardClick: function() {
+        dis.dispatch({
+            action: 'forward_message',
+            content: this.props.mxEvent.getContent(),
```

Comment:
oh, i see we're limiting it to m.room.message. still feels a shame not to spell that out in the params passed here